app/views/posts/index.html.erb

```diff
@@ -1 +1,34 @@
-<h1>List available posts for UserId: <%= @user_id %></h1>
\ No newline at end of file
+<section class="user-profile">
+  <div class="user-card">
+    <img src=<%=@user.photo%> alt="user-profile" />
+    <div class="card-info">
+      <h2><%=@user.name%></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+  <div class="user-posts-container">
+    <% @posts.each_with_index do |post, index| %>
+      <div class="user-post">
+        <%= link_to user_post_path(@user, post) do %>
+          <div class="post-card">
+            <h3>Post #<%=index+1%></h3>
+            <p>
+              <%= post.text.slice(0, 200)... %>
+            </p>
+            <h4>Comments: <%=post.comments_counter%>, Likes: <%=post.likes_counter%></h4>
+          </div>
+        <% end %> 
+        <div class="comments-container">
+          <%if post.comments.empty? %>
+            <p>No comments found!</p>
+          <% else %>
+            <% post.comments.each do |comment| %>
+              <p><%=@user.name%>: <%=comment.text%></p>
+            <% end %>
+          <% end %>
```

Comment:
- [ ] Please make use of the `recent_comments` method in your `Post` model to display the 5 most recent comments on the posts index page 👍 
![image](https://github.com/UbdaNam/Blogiverse/assets/46839548/36fde6e6-2776-4734-8603-bf6a4ff5212b)

Reply:
Thank you for reviewing my PR 👍 . I will implement the changes and request re-review ♻️